quantum/quantum_keycodes.h

```diff
@@ -571,6 +571,43 @@ enum quantum_keycodes {
 #    define SQ_R(n) (n < SEQUENCER_RESOLUTIONS ? SEQUENCER_RESOLUTION_MIN + n : XXXXXXX)
 #    define SQ_T(n) (n < SEQUENCER_TRACKS ? SEQUENCER_TRACK_MIN + n : XXXXXXX)
 
+#endif
+
+#ifdef PROGRAMMABLE_BUTTON_ENABLE
```

Comment:
```suggestion
```

Reply:
Will do.

Could you explain the criteria for some keycodes being always defined, versus others being gated behind `#ifdef`?

Reply:
They're not, this PR should target develop.

Reply:
> They're not, this PR should target develop.

I missed that, sorry.
Will change.